packages/api/src/router/app.ts

```diff
@@ -98,6 +99,26 @@ export const appRouter = createTRPCRouter({
         href: input.href,
       });
     }),
+  createMany: permissionRequiredProcedure
+    .requiresPermission("app-create")
+    .input(validation.app.createMany)
+    .output(z.void())
+    .mutation(async ({ ctx, input }) => {
+      await ctx.db.transaction(async (transaction) => {
```

Comment:
Async transactions do not work because of a limitation in better-sqlite3, instead use the synchronous variant

Reply:
Could you give more insights on this? How am I supposed to run the async functions inside the callback then?

Reply:
You don't, you will need to use synchronous operations in there then. that can be done with `.get()` `.execute()` or something similar with the normal db operations

See https://github.com/WiseLibs/better-sqlite3/issues/32